### ISSUE TITLE
Alloy Mixer not mixing Thermal Alloys

### ISSUE THREAD

**Issue by Fortist** [ZengineeringTeam/Foundry#48]
It seems that despite having everything in, the Alloy Mixer doesnt seem to mix Thermal Expansion special alloys.
I tried all three of Signalum, Enderium, and Lumium and none of them mix.

Electrum and Constantan works, however.
I too have EMBERS: REKINDLED which overtakes this mod's molten forms for the vanilla metals and some common metals like Silver, Lead and Copper. This may or may not factor in this.
Im going to try a crafttweaked recipe and see if that can bypass this.

**Comment by Snownee** [comment]
Thermal Foundation, not Expansion

**Comment by Snownee** [comment]
![image](https://user-images.githubusercontent.com/16059084/51725693-c1366580-209e-11e9-8f53-fdcb316bdb26.png)
![image](https://user-images.githubusercontent.com/16059084/51725696-c85d7380-209e-11e9-8f9e-7e4227c70d6d.png)
![image](https://user-images.githubusercontent.com/16059084/51725761-170b0d80-209f-11e9-813a-8fe9ddae311d.png)

Can't reproduce

**Comment by Fortist** [comment]
Well, You're only showing the it In JEI, which it shows up in for me.
But **IN THE MACHINE,** it doesnt mix. 
[TE is TE regardless of which mod you're talking about in the whole suite, but you're ESL so I'll give you the benefit of the doubt.]

And again, I also have EMBERS which takes over your Molten metals, which could possibly be a reason.
But I just did a workaround and just crafttweaked a fix.
I also added Embers to the "Mod Priorities" list and maybe that could affect it but whatever.
